README.md

```diff
@@ -128,11 +128,19 @@ On an ALP system where the Ansible workload container has been installed,
 using the `install` runlabel, the examples will be available under the
 `/usr/local/share/ansible-container/examples/ansible` directory.
 
-There are three playbooks currently under /usr/local/share/ansible-container/examples/ansible
+There are fourteen playbooks currently under /usr/local/share/ansible-container/examples/ansible
 * playbook.yml
```

Comment:
I would probably break this list up into two or three lists according to type of playbook. The `playbook` and `network` playbooks are example playbooks for general Ansible, and NetworkManager (ALP) specific network configuration.

The `setup_*` playbooks tend to be about setting up workload based services.

The `create_*_vm` playbooks are examples of using a workload service to do real tasks, i.e. create VMs using Libvirt.

Reply:
Divided the list into the above categories